api-tests/tests/helpers.ts

```diff
@@ -48,151 +70,362 @@ export const createDBCluster = async (request, name) => {
         },
       },
     },
-  },
+  }
+  return JSON.parse(JSON.stringify(data))
+}
+
+// Returns DBCluster object for creating 1-node PSMDB cluster
+export const getPSMDBClusterDataSimple = (name: string) => {
+  let data = {
+    apiVersion: 'everest.percona.com/v1alpha1',
+    kind: 'DatabaseCluster',
+    metadata: {
+      name: name,
+    },
+    spec: {
+      backup: {
+        pitr: {
+          enabled: false,
+        },
+      },
+      engine: {
+        type: 'psmdb',
+        replicas: 1,
+        storage: {
+          size: '1Gi',
+        },
+        resources: {
+          cpu: '1',
+          memory: '1G',
+        },
+      },
+      proxy: {
+        type: 'mongos',
+        replicas: 1,
+        expose: {
+          type: 'internal',
+        },
+      },
+      sharding: {
+        configServer: {
+          replicas: 1,
+        },
+        enabled: false,
+        shards: 2,
+      },
+    }
+  }
+  return JSON.parse(JSON.stringify(data))
+}
+
+// Returns DBCluster object for creating 1-node PXC cluster
+export const getPXCClusterDataSimple = (name: string) => {
+  let data = {
+    apiVersion: 'everest.percona.com/v1alpha1',
+    kind: 'DatabaseCluster',
+    metadata: {
+      name: name,
+    },
+    spec: {
+      engine: {
+        type: 'pxc',
+        replicas: 1,
+        config: '[mysqld]\nwsrep_provider_options="debug=1;gcache.size=1G"\n',
+        storage: {
+          size: '1Gi',
+        },
+        resources: {
+          cpu: '1',
+          memory: '1G',
+        },
+      },
+      proxy: {
+        type: 'haproxy',
+        replicas: 1,
+        expose: {
+          type: 'internal',
+        },
+      },
+    },
+  }
+  return JSON.parse(JSON.stringify(data))
+}
```

Comment:
Maybe in the future it will make sense to split these into engine-specific helper files but for now it's okay.